README.md

```diff
@@ -1,33 +1,64 @@
-# HVNC proposals and discussions
+# HGVS Nomenclature
 
-**Under construction**
+This repo is the home of the HGVS Nomenclature maintained by the Human Genome Organization Variation Nomenclature Committee.
```

Comment:
I would consider putting (HUGO) into the text. People use HUGO a lot and oddly may not always know the full meaning.